src/filters.js

```diff
@@ -119,6 +119,7 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        if (str == null) str = '';
```

Comment:
Instead of this, please use https://github.com/atian25/nunjucks/blob/d75a9993c79fb513e9e651bc938f5555213cb2a7/src/filters.js#L6